### PR TITLE
Using Azure Policy to verify the presence of Calico apiserver in arc cluster

### PR DESCRIPTION
**Pull request by jessieziyuantang**
First Jumpstart scenario
Calico #818

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_policy_calico/_index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+type: docs
+title: "Use the Azure Policy Add-on for AKS with a custom Azure Policy to verify whether Calico network policy is enabled on the AKS cluster"
+linkTitle: "Use the Azure Policy Add-on for AKS with a custom Azure Policy to verify whether Calico network policy is enabled on the AKS cluster"
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_policy_calico/_index.md:4]: The title is too long

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_policy_calico/_index.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+weight: 4
+description: >
+---
+
+## Use the Azure Policy Add-on for AKS with a custom Azure Policy to verify whether Calico network policy is enabled on the AKS cluster
+
+The following README will guide you on how to use a custom Azure Policy [Azure Policy for Kubernetes](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/policy-for-kubernetes#:~:text=Azure%20Policy%20extends%20Gatekeeper%20v3,Kubernetes%20clusters%20from%20one%20place.) on an Azure Arc connected Kubernetes Service (AKS) cluster to verify whether Calico network policy is enabled or not.
+
+> **Note: This guide assumes you already deployed an AKS cluster and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion using either [ARM Template](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/aks/aks_arm_template/) or [Terraform](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/aks/aks_terraform/).**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_policy_calico/_index.md:13]: Wondering, why are using AKS as an Arc-connected cluster? Why not CAPI or GKE/EKS?

**Review by likamrat — CHANGES_REQUESTED** [review]
Before continuing with the review, I am a bit confused on why using AKS for this since Azure Policy for Azure Arc-enabled Kubernetes will work with any supported flavor.

Since AKS is a 1st party service, we are not expecting users to Arc-enabled it.

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_policy_calico/_index.md (hunk 1 above)
Comment (jessieziyuantang) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_policy_calico/_index.md:4]: chaneg to:
title: "Use the Azure Policy Add-on to audit Calicocloud / calico enterprise"
linkTitle: "Use the Azure Policy Add-on to create a custom Azure Policy for auditing calicocloud/calico enterprise components"

**Comment by jessieziyuantang** [comment]
I will close this pr and create a new one with eks arc and gke arc.

**Pull request closed without merging**